configs/skeleton/stgcn/metafile.yml

```diff
@@ -13,7 +13,7 @@ Models:
       Architecture: STGCN
       Batch Size: 16
       Epochs: 80
-      FLOPs: 38.2G
+      FLOPs: 3.8G
```

Comment:
Why did the FLOPs value reduce to 1/10?

Reply:
Formally, the reported flops should be calculated using one clip.
![image](https://user-images.githubusercontent.com/58767402/209262664-1b257ccc-bedb-477f-a03b-fac51f201977.png)